src/pages/workspaces/workspace/Notebooks.js

```diff
@@ -69,9 +70,8 @@ const noNotebooksMessage = div({ style: { fontSize: 20 } }, [
 
 class NotebookCard extends Component {
   render() {
-    const { namespace, name, updated, listView, wsName, onRename, onCopy, onDelete, onExport, canWrite } = this.props
-    const tenMinutesAgo = _.tap(d => d.setMinutes(d.getMinutes() - 10), new Date())
-    const isRecent = new Date(updated) > tenMinutesAgo
+    const { namespace, name, timeCreated, updated, listView, wsName, onRename, onCopy, onDelete, onExport, canWrite } = this.props
+    const isRecent = (timeCreated !== updated) && (differenceInMinutes(updated, new Date()) < 10)
```

Comment:
This suppresses the warning for everything. It looks like `timeCreated` gets updated whenever the notebook is saved. We may need to rethink this.

Reply:
Oof, I didn't even think to test for that.